web/src/pages/Service/List.tsx

```diff
@@ -30,6 +31,17 @@ const Page: React.FC = () => {
   const { formatMessage } = useIntl();
   const [rawDataEditorVisible, setRawDataEditorVisible] = useState(false);
   const [rawData, setRawData] = useState({});
+  const [paginationConfig, setPaginationConfig] = useState({ pageSize: 10, current: 1 });
+
+  const savePageList = (page: number, pageSize?: number) => {
+    history.replace(`/service/list?page=${page}&pageSize=${pageSize}`);
+  };
+
+  useEffect(() => {
+    const { page = 1, pageSize = 10 } = querystring.parse(window.location.search);
+    setPaginationConfig({ pageSize: Number(pageSize), current: Number(page) });
+    fetchList({ current: Number(page), pageSize: Number(pageSize) });
```

Comment:
in line112 we already have a `fetchlist` request, it would be better to remove this manually call to avoid call fetchlist twice